src/control/server/ctl_storage_rpc.go

```diff
@@ -29,8 +29,11 @@ import (
 )
 
 const (
-	msgFormatErr      = "instance %d: failure formatting storage, check RPC response for details"
-	msgNvmeFormatSkip = "NVMe format skipped on instance %d as SCM format did not complete"
+	msgFormatErr             = "instance %d: failure formatting storage, check RPC response for details"
+	msgNvmeFormatSkip        = "NVMe format skipped on instance %d"
+	msgNvmeFormatSkipHPD     = msgNvmeFormatSkip + ", use of hugepages disabled in config"
+	msgNvmeFormatSkipFail    = msgNvmeFormatSkip + ", SCM format failed"
+	msgNvmeFormatSkipNotDone = msgNvmeFormatSkip + ", SCM was not formatted"
```

Comment:
```suggestion
	msgNvmeFormatSkipNotDone = msgNvmeFormatSkip + ", SCM was already formatted"
```

Reply:
that Skip message is accurate, it represents the case when no formatting was performed, it could be because SCM was already formatted or because of something else

Reply:
OK, I was understanding that it was only possible if the SCM was previously formatted.
Thanks for the clarification.